core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -31,13 +31,20 @@
     GLOBALS.TAG_REGEX = JSON.parse('{{TAG_REGEX|js_string}}');
   </script>
 
-  <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+  <!-- Updated previous version to current version of google charts
+  https://developers.google.com/chart/interactive/docs/basic_load_libs#update-library-loader-code -->
+  <script type="text/javascript" async src="https://www.gstatic.com/charts/loader.js"></script>
   <script type="text/javascript">
-    if (window.google && window.google.load) {
-      google.load('visualization', '1', {packages: ['corechart']});
-    } else {
-      throw 'error: Could not load google visualization library. Are you offline?';
-    }
+    function onloadChartLoadHandler() {
```

Comment:
I'm a bit confused. Where's the closing } for this function?

Also, if you have a function, does this even get called at all?